Syntax error in BPF script

When running heap-alloc-flames I get the error
```
/virtual/main.c:49:37: error: expected ';' after top level declarator
BPF_STACK_TRACE(stack_traces, 10240)
                                    ^
                                    ;
1 error generated.
```
Adding the semicolon to src/heap/heap_profile.py appears to fix this issue.

--- a/src/heap/heap_profile.py
+++ b/src/heap/heap_profile.py
@@ -87,7 +87,7 @@ BPF_HASH(counts, struct key_t);
 BPF_HASH(start, u32);
 BPF_HASH(sample_count, u64);
 BPF_HASH(tids, long);
-BPF_STACK_TRACE(stack_traces, 10240)
+BPF_STACK_TRACE(stack_traces, 10240);
 
 
 int trace_alloc(struct pt_regs *ctx, long tid, char *name, int nameLength, int wordSize) {

